tests/bits/periodic_flux_coupling_01.output

```diff
@@ -1,5 +1,5 @@
 
 DEAL::16
 28
-52
```

Comment:
is this change expected?

Reply:
Let's say that I am not surprised that this change refines a few more cells in serial.
The purpose of the test is to check that creating an appropriate sparsity pattern works.

Reply:
Old:

![grid_old](https://user-images.githubusercontent.com/2423533/43898375-ebdee98c-9bde-11e8-8436-a6395f954656.png)

New:

![grid_new](https://user-images.githubusercontent.com/2423533/43898378-f01b07ce-9bde-11e8-80d4-4c8ed3ea9b6a.png)

Reply:
Nice!